cmd/rcd/rcd.go

```diff
@@ -32,18 +41,209 @@ See the [rc documentation](/rc/) for more info on the rc flags.
 		if rcflags.Opt.Enabled {
 			log.Fatalf("Don't supply --rc flag when using rcd")
 		}
+
 		// Start the rc
 		rcflags.Opt.Enabled = true
 		if len(args) > 0 {
 			rcflags.Opt.Files = args[0]
 		}
+
+		if rcflags.Opt.WebUI {
+
+			// Get the latest release details
+			WebUIURL, tag, size := GetLatestReleaseURL()
+
+			// Load the file
+			exists := exists(tag + ".zip")
+			if !exists {
+				fmt.Println("Downloading webui binary. Please wait Size :" + strconv.Itoa(size))
+				if err := DownloadFile(tag+".zip", WebUIURL); err != nil {
+					panic(err)
+				} else {
+					println("Unzipping")
+					if err := os.RemoveAll("/webui"); err != nil {
+						fmt.Println("No previous downloads to remove")
+					}
+					if err := Unzip(tag+".zip", "webui"); err != nil {
+						panic("Error extracting file")
+					}
+				}
+			} else {
+				println("Files already exists. Skipping download")
+			}
+		}
+
 		s, err := rcserver.Start(&rcflags.Opt)
 		if err != nil {
 			log.Fatalf("Failed to start remote control: %v", err)
 		}
 		if s == nil {
 			log.Fatal("rc server not configured")
 		}
+
 		s.Wait()
 	},
 }
+
+/*
+GetLatestReleaseURL returns the latest release details of the rclone-webui-react
+*/
+func GetLatestReleaseURL() (string, string, int) {
+	resp, err := http.Get("https://api.github.com/repos/negative0/rclone-webui-react/releases/latest")
+	if err != nil {
+		panic("Error getting latest release of rclone-webui")
+	}
+	results := GitHubRequest{}
+	if err := json.NewDecoder(resp.Body).Decode(&results); err != nil {
+		panic("Could not decode results from http request")
+	}
+
+	res := results.Assets[0].BrowserDownloadURL
+	tag := results.TagName
+	size := results.Assets[0].Size
+	//fmt.Println( "URL:" + res)
+
+	return res, tag, size
+
+}
+
+/*
+DownloadFile is a helper function to download a file from url to the filepath
+*/
+func DownloadFile(filepath string, url string) error {
```

Comment:
I'd be tempted to use some of rclone's internal functions to do this, but that can wait for another update!